apps/frontend/src/libs/components/sidebar/sidebar-item/sidebar-item.tsx

```diff
@@ -5,15 +5,21 @@ import { type NavigationItem } from "~/libs/types/types.js";
 
 import styles from "./styles.module.css";
 
-const SidebarItem = ({ href, icon, label }: NavigationItem): JSX.Element => (
-	<li className={styles["sidebar-item"]}>
-		<NavLink className={styles["navigation-link"] ?? ""} to={href}>
-			<span className={styles["navigation-icon"]}>
-				<Icon height={20} name={icon} width={20} />
-			</span>
-			<label className={styles["navigation-item-label"]}>{label}</label>
-		</NavLink>
-	</li>
-);
+const getClassName = ({ isActive }: { isActive: boolean }): string => {
+	return `${styles["navigation-link"] ?? ""} ${isActive ? (styles["active"] ?? "") : ""}`;
+};
+
+const SidebarItem = ({ href, icon, label }: NavigationItem): JSX.Element => {
+	return (
+		<li className={styles["sidebar-item"]}>
+			<NavLink className={getClassName} to={href}>
```

Comment:
Remove all `import { NavLink } from "react-router-dom";` from the project
we need to use our `Link` component wrapper. You can use `NavLink` from `react-router-dom` in our Link component

Reply:
But how then change style of Link component? It was said that we cannot pass classname to Link component

Reply:
Let's leave it as it is in this PR, will create a separate ticket when we decide on approach